### ISSUE TITLE
[UI] Make use of the Warning component for notes & warnings

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster#1388]
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

Continuation of https://github.com/taskcluster/taskcluster/issues/1207.

There are a couple of areas in the documentation where we should be using the `Warning` component to inform users of a certain note or warning. I can think of two examples:

1. https://taskcluster-ui.herokuapp.com/docs/manual/tasks/workertypes

> NOTE: RFC#145 addresses this confusion by introducing the term taskQueueId to name the queue in which a task is stored. This RFC is not yet fully implemented, so the Queue API still uses identifiers provisionerId and workerType.

2. https://taskcluster-ui.herokuapp.com/docs/manual/tasks/priority

> Note that this can result in starvation: if there are enough tasks at a given priority to keep all available workers busy, then no tasks of lower priority will run.

**Comment by vishakhanihore** [comment]
I want to work on this

**Comment by djmitche** [comment]
All yours!  

We want to make sure everyone gets a chance to work on an issue, and we don't have too many!  So once this is done, please look at other issues, not labeled "outreachy", and find one that looks good.  Those are likely to be a bit more challenging.
